tests/python/test_bit_array_vectorization.py

```diff
@@ -93,3 +93,85 @@ def verify(dx: ti.template(), dy: ti.template()):
     verify(0, -1)
     assign_vectorized(-1, 0)
     verify(-1, 0)
+    assign_vectorized(1, 1)
+    verify(1, 1)
+    assign_vectorized(1, -1)
+    verify(1, -1)
+    assign_vectorized(-1, -1)
+    verify(-1, -1)
+    assign_vectorized(-1, 1)
+    verify(-1, 1)
+
+
+@ti.test(require=ti.extension.quant)
```

Comment:
```suggestion
@ti.test(require=ti.extension.quant, debug=True)
```
Debug mode needed for `assert` to work.